### ISSUE TITLE
Handle problems with solc-bin more gracefully

### ISSUE THREAD

**Issue by cameel** [ethereum/remix-project#1885]
Currently, when there's an issue with solc-bin (like the recent #1882), Remix silently ignores the error and just presents an empty list of compiler versions to the user. I think it should instead show an error message somewhere to make it clear that something's broken. The message should give some details about what the issue is so that it can be reported in a useful way. E.g. if `binaries.soliditylang.org` is down, it should say that. It might also be a good idea to direct users to report it in the Solidity repository if the problem persists.

**Comment by Aniket-Engg** [comment]
I think, in that case, Remix showed this message: `Cannot load compiler version list. It might have been blocked by an advertisement blocker. Please try deactivating any of them from this page and reload.`

But, yes It can definitely be improved to suggest more possible problems behind this.

**Comment by cameel** [comment]
Fair enough. In this case I did not run into this myself and I'm just requesting this based on what other users reported. They must have just not noticed it so maybe it would be enough to make it more prominent?
